name-service/js/package.json

```
@@ -51,5 +61,16 @@
     "@solana/web3.js": "^1.11.0",
     "bn.js": "^5.1.3",
     "borsh": "^0.4.0"
+  },
+  "mocha": {
+    "require": [
+      "ts-node/register"
+    ],
+    "recursive": true,
+    "extension": [
+      "ts",
+      "js",
+      "tsx"
+    ]
```

nit: Let's keep this simple since everything is in ts
```suggestion
    "extension": [
      "ts",
    ]
```